src/hostapi/coreaudio/pa_mac_core_utilities.h

```diff
@@ -111,9 +110,66 @@
 # define VVDBUG(MSG)
 #endif
 
+#if (MAC_OS_X_VERSION_MIN_REQUIRED >= MAC_OS_X_VERSION_10_5)
+#define PA_NEW_HAL 1
+#else
+#define PA_NEW_HAL 0
+#endif
 
+#if PA_NEW_HAL
+typedef AudioObjectPropertyListenerProc pa_AudioDevicePropertyListenerProc;
+#else
+typedef AudioDevicePropertyListenerProc pa_AudioDevicePropertyListenerProc;
+#endif
 
-
+OSStatus pa_AudioHardwareGetProperty(
+    AudioHardwarePropertyID inPropertyID,
```

Comment:
Continuation lines shouldbe indented by 8 or line up with the "(".

Reply:
Never mind. I see the whole file does this. We can address that later. Better to be consistent for now.